README.md

```diff
@@ -1,13 +1,15 @@
 ## Running ASP.NET and ASP.NET Core in the same IIS pipeline
 
-This is an experiment showing how to run ASP.NET WebAPI and ASP.NET Core in the IIS pipeline (it can be in process or out of process). The idea is to experiment with ways to use any custom logic to determine which routes go to ASP.NET Core and which ones go to ASP.NET. This can aid is porting large applications to ASP.NET Core in a route by route manner.
+This is an experiment showing how to run ASP.NET WebAPI and ASP.NET Core in the IIS pipeline (it can be in process or out of process). The idea is to experiment with ways to use any custom logic to determine which routes go to ASP.NET Core and which ones go to ASP.NET. This can aid in porting large applications to ASP.NET Core in a route by route manner.
 
 ### Instructions
 
-1. Make sure IIS is running in 64 bit more for ASP.NET.
+1. Make sure IIS is running in 64 bit mode for ASP.NET.
 
     ![image](https://user-images.githubusercontent.com/95136/79828724-2df9e780-8356-11ea-9890-7e478c87b86d.png)
 
+1. Mark Greenfield as the startup project and run the app for first time initialization with IIS Express.
```

Comment:
There seems to be some 1st time initialization that needs to be done in the `.vs` folder when you launch the app for the first time.

![image](https://user-images.githubusercontent.com/23503973/137009582-77428e39-6229-4093-9631-7a67ace0ad31.png)